src/test/java/uk/gov/companieshouse/extensions/api/reasons/ReasonServiceUnitTest.java

```diff
@@ -192,8 +187,8 @@ public void testReasonIsRemovedFromRequest() {
 
         FileTransferApiClientResponse response = new FileTransferApiClientResponse();
         response.setHttpStatus(HttpStatus.NO_CONTENT);
-        when(fileTransferApiClient.delete("1234")).thenReturn(response);
-        when(fileTransferApiClient.delete("5678")).thenReturn(response);
+        lenient().when(fileTransferApiClient.delete("1234")).thenReturn(response);
```

Comment:
again, i'm worried we've broken something if these calls were executed correctly before using lenient

Reply:
See above

Reply:
doReturn(response).when(fileTransferApiClient).delete("1234");
        doReturn(response).when(fileTransferApiClient).delete("5678");

Reply:
Updated this to use doReturn so as to avoid the side effect of real method invocation.